SubEthaEdit-Mac/Source/TabbedDocument.h

```diff
@@ -0,0 +1,34 @@
+//
+//  TabbedDocument.h
+//  SubEthaEdit
+//
+//  Created by Francisco Tolmasky on 8/30/19.
+//  Copyright © 2019 SubEthaEdit Contributors. All rights reserved.
+//
+
+#import <Cocoa/Cocoa.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+typedef void (^AlertConsequence)(__kindof NSDocument *, NSModalResponse);
+
+@interface TabbedDocument : NSDocument
+
+@property (readonly) BOOL hasAlerts;
+
+- (void)alert:(NSString *)message
+        style:(NSAlertStyle)style
+      details:(NSString *)details
+      buttons:(NSArray *)buttons
+         then:(nullable AlertConsequence)then;
+
+- (void)inform:(NSString *)message details:(NSString *)details;
+
+- (void)warn:(NSString *)message
+     details:(NSString *)details
+     buttons:(NSArray *)buttons
+        then:(nullable AlertConsequence)then;
+
+@end
+
+NS_ASSUME_NONNULL_END
```

Comment:
please remove the NS_ASSUME_NONULL_ macros as well

Reply:
Fixed: https://github.com/tolmasky/SubEthaEdit/commit/b7298103c3f3631d6fb549829b8969ec2b1b50fe